### ISSUE TITLE
::uism/error-event does not work on loads

### ISSUE THREAD

**Issue by psagers** [fulcrologic/fulcro#376]
In trying to use `::uism/error-event` in the `uism/load` options, I was just getting console errors:

```
WARN [com.fulcrologic.fulcro.ui-state-machines:557] -  A fallback occurred, but no event was defined by the client. Sending generic ::uism/load-error event.
DEBUG [com.fulcrologic.fulcro.ui-state-machines:127] -  Triggering :com.fulcrologic.fulcro.ui-state-machines/load-error on null with null
ERROR [com.fulcrologic.fulcro.ui-state-machines:739] -  Attemped to trigger event  :com.fulcrologic.fulcro.ui-state-machines/load-error on state machine null , but that state machine has not been started (call begin! first).
```

Clearly the error handler has lost track of both the event-id and asm-id. Looking at the code, it appears that the [fallback mutation](https://github.com/fulcrologic/fulcro/blob/44dabadd62f29a6f7f2a2f33a1b92bbafd392d6c/src/main/com/fulcrologic/fulcro/ui_state_machines.cljc#L562) is looking for its data in `(-> env :load-request :post-mutation-params)`, but there's no `:load-request` key in `env`.

[11.3.6. Server Error Demo](http://book.fulcrologic.com/#_server_error_demo) indicates that the original request is actually under the `:load-params` key of the fallback mutation params. The following change resolved the issue for me:

```diff
-(defmutation handle-load-error [_]
-  (action [{:keys [app load-request]}]
-    (handle-load-error* app load-request)))
+(defmutation handle-load-error [params]
+  (action [{:keys [app]}]
+    (handle-load-error* app (:load-params params))))
```

The tests appear to be too narrow to catch this at the moment. (I haven't prepared a pull request, in part because I haven't figured out how to run the tests yet.)

**Comment by awkay** [comment]
Hey Peter. You can run the tests in various ways. `make` will run all of them once. 

`make dev` will run a watch on CLJ tests.

Running a shadow-cljs watch on the test build will create a live runner for cljs tests:
`shadow-cljs watch test` that can be seen on localhost:9002

**Comment by awkay** [comment]
You're right, this logic can't be right...guess I missed a test for that one.

**Comment by awkay** [comment]
I agree with your fix. I'll just patch it and post a release after playing with it a little.

**Comment by awkay** [comment]
Fixed in 3.1.14
